app/routes/bdb/components/BdbDetail.js

```diff
@@ -181,7 +185,7 @@ export default class BdbDetail extends Component<Props, State> {
     const events =
       companyEvents &&
       companyEvents
-        .sort((a, b) => new Date(b.startTime) - new Date(a.startTime))
+        .sort((a, b) => Date.parse(b.startTime) - Date.parse(a.startTime))
```

Comment:
How does this help when you get the first events for a certain company, not the newest ones?

Reply:
With pagination you mean? Isn't events sorted backend so that you get the 30 most recent events? The alternative wouldn't make much sense

Reply:
When looking at BEKK in prod, I get the "first" 30 events from BEKK, not the 30 most recent ones. Looks like the backend sorts by pk, not by date.

Reply:
Surely the events from the generic `/events` endpoint are sorted by date with the most recent first, right? Otherwise the front page and the calendar wouldn't do their job at all. So I guess that means the queryset isn't sorted the same way when you add a filter..?